manifest_staging/charts/eraser/templates/imagejobs.eraser.sh-customresourcedefinition.yaml

```diff
@@ -115,3 +115,50 @@ spec:
     storage: false
     subresources:
       status: {}
+  - name: v1alpha2
```

Comment:
why do we have v1alpha2 of imagejob and imagelists?

Reply:
We shouldn't

Reply:
I think re-running codegen and manifest gen should fix this, since those files were removed.